app/controllers/home_controller.rb

```diff
@@ -47,6 +47,16 @@ def dashboard
     end
   end
 
+  def switch_on_translation
```

Comment:
maybe we should name this
```suggestion
  def switch_on_translation_helper
```

Is this only for development mode? If so, couldn't we use an existing route somehow -- https://github.com/publiclab/plots2/blob/main/test/functional/user_tags_controller_test.rb#L8-L15 maybe? To add the tag? though i guess it's 2 steps.